docs/time/performance.inc.rst

```diff
@@ -10,3 +10,52 @@ Performance Tips
 
 Here we provide some tips and tricks for how to optimize performance of code
 using `astropy.time`.
+
+Light Travel Times
+------------------
+
+Listed below are two approaches to calculating light travel times for tens of
+thousands of sources in a degree patch of the sky. The first approach calculates
+the travel times without iteration while the second calculates using iteration.
+
+
+>>> import numpy as np
+>>> import astropy.coordinates as coord
+>>> import astropy.units as u
+>>> from astropy.time import Time
+
+>>> ra = np.random.normal(0.0, 1.0, 50000)
+>>> dec = np.random.normal(0.0, 1.0, 50000)
+>>> coos = coord.SkyCoord(ra, dec, unit=u.deg)
+>>> observatory = coord.EarthLocation.from_geocentric(5327448.9957829, -1718665.73869569, 3051566.90295403, unit='m') #Lapalma Observatory
+
+Approach Not Using Iteration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>>> Time.light_travel_time(coos, location=observatory)
+
+   **CPU times:** user 56 ms, sys: 3.21 ms, total: 59.2 ms
+
+   **Wall time:** 58.2 ms
+
+Approach Using Iteration
+^^^^^^^^^^^^^^^^^^^^^^^^
+>>> ltts = [Time.light_travel_time(coo, location=coos.location) for coo in coos]
+
+   **CPU times:** user 16min 45s, sys: 5.08 s, total: 16min 50s
+
+   **Wall time:** 16min 58s
+
+As you can see, the user time is significantly longer.
```

Comment:
Perhaps explain why? Iterating over a ```SkyCoord``` is not fast, and the former approach also takes advantage of vectorised operations in numpy...

Reply:
Sure, no problem!